README.md

```diff
@@ -24,6 +24,10 @@ For browsers that need it, there are also some minor polyfills included:
 - [`URL constructor`](https://github.com/webcomponents/URL)
 
 ## How to use
+### Install polyfills
+```bash
+npm install @webcomponents/webcomponentsjs
```

Comment:
Right now this requires `npm install @webcomponents/webcomponentsjs@next`. Should we do that until it's tagged as `latest`?

Reply:
Also, for elements, this should have the `-D` flag...

Reply:
This is for the v2 branch, which I think won't be merged to master till this is published next week.